modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-blocking-nodeport-access_{context}"]
+= Blocking external access to the NodePort service on a specific host interface
+
+OVN-Kubernetes does not restrict the host interface where a NodePort service can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
+
+.Procedure
+. Change the NODEPORT variable to the host port number assigned to your kubernetes NodePort service 
++
+[source,terminal]
+----
+$ export NODEPORT=30700
+----
+. Change the INTERFACE_IP value to the IP address from the host interface that you want to block. 
++
+[source,terminal]
+----
+$ export INTERFACE_IP=192.168.150.33
+----
+. Insert a new rule in the `nat` table PREROUTING chain to drop all packets that match the destination port and ip. 
++
+[source,terminal]
+----
+$ sudo nft -a insert rule ip nat PREROUTING tcp dport $NODEPORT ip daddr $INTERFACE_IP drop
+----
+
+. List the new rule with the following command:
++
+[source,terminal]
+----
+$ sudo nft -a list chain ip nat PREROUTING
+table ip nat {
+	chain PREROUTING { # handle 1
+		type nat hook prerouting priority dstnat; policy accept;
+		tcp dport 30700 ip daddr 192.168.150.33 drop # handle 134
+		counter packets 108 bytes 18074 jump OVN-KUBE-ETP # handle 116
+		counter packets 108 bytes 18074 jump OVN-KUBE-EXTERNALIP # handle 114
+		counter packets 108 bytes 18074 jump OVN-KUBE-NODEPORT # handle 112
+	}
+}
+----
++
+[NOTE]
+====
+Note your `handle` number of the newly added rule. You will need the number for removal in the following step. 
```

Comment:
```suggestion
Note your `handle` number of the newly added rule. You need to remove `handle` number in the following step.
```
You could also pull this up into the step above. "List the new rule by running the following command. Note your `handle` number ..."